### PR TITLE
light-blinkt: add documentation for new light

### PR DESCRIPTION
**Pull request by imrehg**
**Description:**

Add documentation for new light, using [Blinkt!](https://shop.pimoroni.com/products/blinkt).

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#7377

### CODE REVIEW

File: source/_components/light.blinkt.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: "Blinkt!"
+description: "Instructions how to setup Sense HAT LED lights within Home Assistant."
```
Comment (Landrash) [inline source/_components/light.blinkt.markdown]: Description differs from rest of the document.

File: source/_components/light.blinkt.markdown [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: "Blinkt!"
```
Comment (Landrash) [inline source/_components/light.blinkt.markdown]: Remove the `!` from the title.

File: source/_components/light.blinkt.markdown (hunk 1 above)
Comment (imrehg) [inline source/_components/light.blinkt.markdown]: Sorry, that's very careless of me, fixing.

File: source/_components/light.blinkt.markdown (hunk 2 above)
Comment (imrehg) [inline source/_components/light.blinkt.markdown]: @Landrash The component's original name actually includes the `!` (see https://shop.pimoroni.com/products/blinkt), of course I'm happy to remove it, just wondering if there's a specific reason to do so? (does it mess with these docs some way?)

Comment (Landrash) [inline source/_components/light.blinkt.markdown]: Oh if that's the name standardized the other way around and try to use Blinkt! everywhere possible. 😊

Comment (Landrash) [inline source/_components/light.blinkt.markdown]: I just realized that's what you did. Good work.

**Comment by balloob** [comment]
Cherry-picked for 0.44

**Pull request merged**